prom-keycloak-proxy/overlays/nerc-ocp-obs/deployments/prom-keycloak-proxy/deployment.yaml

```
@@ -0,0 +1,99 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: prom-keycloak-proxy
+    deployment: prom-keycloak-proxy
+  name: prom-keycloak-proxy
+  namespace: prom-keycloak-proxy
+spec:
+  replicas: 1
+  revisionHistoryLimit: 0
+  selector:
+    matchLabels:
+      app: prom-keycloak-proxy
+      deployment: prom-keycloak-proxy
+  strategy:
+    activeDeadlineSeconds: 21600
+    recreateParams:
+      timeoutSeconds: 600
+    resources: {}
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: prom-keycloak-proxy
+        deployment: prom-keycloak-proxy
+      name: prom-keycloak-proxy
+    spec:
+      containers:
+        - env:
+            - name: PROXY_AUTH_CLIENT_ID
+              valueFrom:
+                secretKeyRef:
+                  key: PROXY_AUTH_CLIENT_ID
+                  name: prom-keycloak-proxy-secrets
+            - name: PROXY_AUTH_CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  key: PROXY_AUTH_CLIENT_SECRET
+                  name: prom-keycloak-proxy-secrets
```

If the key in the Secret matches the environment variable name, you can replace these with an `envFrom` directive:

```suggestion
        - envFrom:
            secretRef:
              name: prom-keycloak-proxy-secrets
          env:
```

---

I applied this suggestion, thanks.